code/style.css

```diff
@@ -2,6 +2,139 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+body {
+  display: flex;
+}
+
+header {
+  display:flex;
+  width: 100%;
+  height: auto;
+  text-align: center;
+  background-color:rgba(40, 233, 207, 0.753);
+  color:white;
+  position:fixed;
+}
```

Comment:
This is where you would style your background image. Put it as `background-image: url(yourimage.jpg);`

You can also make it fit to size and scale with the browser. Here you can read some more on it:
https://www.w3schools.com/cssref/pr_background-image.asp